### PR TITLE
ISSUE-666 Delete user data: auto-complete contacts

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

File: docs/modules/ROOT/pages/tmail-backend/webadmin.adoc [hunk 1]
```diff
@@ -576,3 +576,59 @@ Valid status includes:
 - `IN_PROGRESS`: Currently executed
 - `FAILED`: Error encountered while executing this step. Check the logs.
 - `ABORTED`: Won't be executed because of previous step failures.
+
+=== Delete data of a user
```
Comment (chibenwa) [inline docs/modules/ROOT/pages/tmail-backend/webadmin.adoc:580]: IMO not needed: we already documented this endpoint on James side

**Review by vttranlina — APPROVED** [review]

File: docs/modules/ROOT/pages/tmail-backend/webadmin.adoc (hunk 1 above)
Comment (Arsnael) [reply docs/modules/ROOT/pages/tmail-backend/webadmin.adoc]: Not even saying there is an extra step? Can at least point to the james doc and say there is an extra step to it with tmail?

**Comment by Arsnael** [comment]
```
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:4.6.1:compile (scala-compile-first) on project jmap-extensions: Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:4.6.1:compile failed: Compilation failed: InterfaceCompileFailed
```
But I guess it's because it's still point to james v 3.8.0-SNAPSHOT for now^^'

File: docs/modules/ROOT/pages/tmail-backend/webadmin.adoc (hunk 1 above)
Comment (quantranhong1999) [reply docs/modules/ROOT/pages/tmail-backend/webadmin.adoc:580]: rollbacked the docs. A bit redundant is better than missing IMO...

**Review by Arsnael — APPROVED** [review]

**Pull request merged**